dev/Makefile

```diff
@@ -1,6 +1,48 @@
 dev:
 	docker-compose -f docker-compose.yml up
-.PHONY: dev
+
+# Build or rebuild docker containers.
+build-dev:
+ifeq (,$(wildcard /usr/bin/yarn))
+	cd ../web; \
+	sudo apt-get update && sudo apt-get install -y yarn || \
+	apt-get update && sudo apt-get install -y yarn || \
+	sudo pacman --needed --noconfirm -S yarn || \
+	pacman --needed --noconfirm -S yarn; \
+	yarn install && \
+	yarn build; \
+	cd ../dev
+endif
+	docker-compose -f docker-compose.yml build
+
+# Use this target instead of `dev` to support VPN 
+# configs (OpenVPN or other).
+dev-ext:
```

Comment:
i don't really see a drawback in always using an external network - could be a better practice anyway. so maybe we can just have the regular dev target do this.

OTOH, i'd rather avoid duplicating the docker-compose (which had already been changed :))